docs/usage/collect-metrics.md

```diff
@@ -0,0 +1,31 @@
+# Collect metrics
+
+This section presents the metrics exposed by **liqo components**, which can be scraped by [Prometheus](https://prometheus.io/). In order to show how to use those metrics some [Grafana](https://grafana.com/) dashboards are provided.
+
+## Prerequisites
+
+In order to collect metrics from Liqo components, you need to install Prometheus and Grafana.
+
+It's suggested to install the **kube-prometheus** stack to deploy them. Instructions can be found [here](https://github.com/prometheus-operator/kube-prometheus)
```

Comment:
I wouldn't say you have to install Prometheus, as if you care about observability it is likely you already have it installed. You could rephrase saying that you assume it to be present, similarly to what is done [here](https://cert-manager.io/docs/usage/prometheus-metrics/) for instance.